admin/metabox/class-metabox.php

```diff
@@ -72,6 +74,7 @@ public function __construct() {
 	 * @return bool Whether the request comes from an IE 11 browser.
 	 */
 	public static function is_internet_explorer() {
+		return true;
```

Comment:
@afercia I'm guessing this should not have been committed?

Reply:
I thought I did remove that 😄